com.unity.formats.alembic/Runtime/Scripts/Importer/AlembicStreamPlayer.cs

```diff
@@ -243,20 +243,22 @@ bool InitializeAfterLoad()
             StreamDescriptor.MediaStartTime = (float)start;
             StreamDescriptor.MediaEndTime = (float)end;
 
-            var defaultMat = AlembicMesh.GetDefaultMaterial();
-
-            foreach (var meshRenderer in gameObject.GetComponentsInChildren<MeshRenderer>(true))
+            if (reloadMaterials)
             {
-                var mats = meshRenderer.sharedMaterials;
-                for (var i = 0; i < mats.Length; ++i)
+                var defaultMat = AlembicMesh.GetDefaultMaterial();
+                foreach (var meshRenderer in gameObject.GetComponentsInChildren<MeshRenderer>(true))
                 {
-                    if (mats[i] == null) // Add the default material if no material present
```

Comment:
This code changed in the last 2-3 weeks.
Are you sure you still need your changes ?
Basically this null check here assures that only null material references are replaced with default materials (this is to prevent default magenta materials).

Reply:
Ok, indeed it should also solve the issue. The latest official package didn't had that so I didn't saw that when I was making the changes, but seems a much more simple and effective way. I will test the dev branch tomorrow on our codebase and see if it works. Thanks @clusty.

Reply:
Did a quick test, and it's working. When do you expect that this will be available on the package manager as either preview or release?

This PR can be close, no need for any change.